distributions/aws/examples/rds-s3/README.md

```diff
@@ -77,19 +102,16 @@ Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aw
           minioServiceHost=s3.amazonaws.com
           minioServiceRegion=us-west-2
           ```
-    1. Configure `apps/pipeline/upstream/env/aws/secret.env` file with your RDS database username and password that were configured when following the steps in Create RDS Instance.
-        - For example, if your username is `admin` and your password is `Kubefl0w` then your `secret.env` file should look like:
```

Comment:
oh didnt notice this earlier. we need this line(for example,...) because customer might change the default

Reply:
added the example line back in